hack/install_kustomize.sh

```diff
@@ -123,31 +134,38 @@ s390x)
     ;;
 esac
 
-releases=$(curl -s $release_url)
+releases=$(curl -s "$release_url")
 
 if [[ $releases == *"API rate limit exceeded"* ]]; then
   echo "Github rate-limiter failed the request. Either authenticate or wait a couple of minutes."
   exit 1
 fi
 
-RELEASE_URL=$(echo "${releases}" |\
-  grep browser_download.*${opsys}_${arch} |\
-  cut -d '"' -f 4 |\
-  sort -V | tail -n 1)
+RELEASE_URL="$(find_release_url "$releases" "$opsys" "$arch")"
+
+if [[ "$arch" == "arm64" ]] && [[ -z "$RELEASE_URL" ]] ; then
+    # fallback to the old behavior of downloading amd64 binaries on aarch64 systems.
+    # People might have qemu-binfmt-misc installed, so it worked for them previously.
+    echo "Version $version does not exist for ${opsys}_arm64, trying ${opsys}_amd64 instead."
```

Comment:
nit: to match the formatting used at L157:

```suggestion
    echo "Version $version does not exist for ${opsys}/arm64, trying ${opsys}/amd64 instead."
```

Reply:
done, and rebased onto latest master :) LMK if that fixes the CI.